pyproject.toml

```diff
@@ -272,7 +272,7 @@ parameterized = ">=0.7.4"
 idna = ">=2.5"
 
 # The following are used by the release script
-click = "==8.1.0"
+click = "==8.1.1"
```

Comment:
Sorry, this'll be my fault for making scripts-dev by checked by mypy.

(Not sorry)